sriov_antrea/sriov_antrea_ci_start.sh

```diff
@@ -25,6 +25,8 @@ export KUBECONFIG=${KUBECONFIG:-/etc/kubernetes/admin.conf}
 export SRIOV_INTERFACE=${SRIOV_INTERFACE:-auto_detect}
 export VFS_NUM=${VFS_NUM:-4}
 
+antrea_scm_dir="/jenkins/workspace/antrea_ci/PR/${ANTREA_CNI_PR}"
```

Comment:
we should discuss if there are any security implications to this.

i think its the same as normal PR we run on our BM servers, but i think its worth a short discussion in one of our standups. 

@AbdYsn can you rasie it ?